verilog/tools/kythe/kythe_facts.cc

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+#include <string>
+
+#include "absl/strings/substitute.h"
+#include "verilog/tools/kythe/kythe_facts.h"
+
+namespace verilog {
+namespace kythe {
+
+std::string VName::ToString() const {
+  return absl::Substitute(
+      R"({"signature": "$0","path": "$1","language": "$2","root": "$3","corpus": "$4"})",
+      signature_base_64, path, language, root, corpus);
```

Comment:
I think this is simpler this way just for making the output a little readable in case of `absl::StrCat` the `"` will make it not looking good and the `,` also.